### PR TITLE
COMMERCE-4629 Run tests on CI

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by nikki-pru** [comment]
@riccardo-alberti , the functional tests were not supposed to run when running `ci:test:commerce`. I'm fixing that on https://issues.liferay.com/browse/LRQA-61552

**Comment by riccardo-alberti** [comment]
@nikki-pru thanks... i included your change in this PR. I restored the CommerceOrderIndexerTest because i need to see if it works on CI.

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:stop

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:stop

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:stop

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by riccardo-alberti** [comment]
ci:test:commerce

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
Running relevant tests. If pass we can forward

**Comment by riccardo-alberti** [comment]
tests are passing the PR is ready

File: modules/apps/commerce/commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/pricing/test/CommercePriceListLowestDiscoveryTest.java [hunk 1]
```diff
@@ -113,6 +115,12 @@ public void setUp() throws Exception {
 			_group.getGroupId(), _commerceCurrency.getCode());
 	}
 
+	@After
+	public void tearDown() throws Exception {
```
Comment (igorbeslic) [inline modules/apps/commerce/commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/pricing/test/CommercePriceListLowestDiscoveryTest.java:119]: @riccardo-alberti @marco-leo shouldn't it be better to have base class that would hold this copy/pasted code?? That is why we have OOP

File: modules/apps/commerce/commerce-product-test/src/testIntegration/java/com/liferay/commerce/product/service/test/CPInstanceLocalServiceTest.java [hunk 2]
```diff
@@ -81,6 +83,20 @@ public void setUp() throws Exception {
 			ServiceContextTestUtil.getServiceContext(_company.getGroupId()));
 	}
 
+	@After
+	public void tearDown() throws Exception {
```
Comment (igorbeslic) [inline modules/apps/commerce/commerce-product-test/src/testIntegration/java/com/liferay/commerce/product/service/test/CPInstanceLocalServiceTest.java]: @marco-leo do you do architecture review??? Again copy/pasted code

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**